content/docs/policy/approval/approver-policy/README.md

```diff
@@ -271,6 +271,43 @@ spec:
   ...
 ```
 
+#### Validations
+
+Since approver-policy 0.11.0 it is now possible to define more advanced allowed
+request attribute validation rules using
+[Common Expression Language (CEL)](https://kubernetes.io/docs/reference/using-api/cel/).
+The main motivation for adding this feature, was to enable allowed request
+attribute values as a function of the request namespace. But since CEL is a
+small programming language, validation rules can also complement or replace the
+basic wildcard support in allowed attribute value specification.
```

Comment:
```suggestion
basic wildcard support in `allowed` attribute value specification.
```
?

Reply:
Unsure about this. In this context, it's not the code term `allowed` we are addressing - I think. Leaving the decision up to the next reviewer. 😉